www/public/index.html

```diff
@@ -41,8 +55,241 @@ <h1>Work in Progress</h1>
     <li><a href="https://youtu.be/ZnYa99QoznE?t=4790">Roc at the Berlin FP Meetup</a> - September 1, 2020 (overall
       vision for the language)</li>
   </ul>
-  <p>Roc also compiles to WebAssembly and you can try it out on the web <a href="repl/">here</a> (remembering it's not ready yet!)</p>
-  <p>You can check out the <a href="https://github.com/roc-lang/roc">source code repository</a> if you'd like to play around with it in its current state.</p>
+
+  <h2>A <em>Fast</em> Language</h2>
+
+  <h3>Goal</h3>
+
+  <p>The goal is that Roc runs so fast that in practice, you wouldn't expect a significant
+    performance boost from switching to a more popular language unless that language runs
+    at least as fast as C.</p>
```

Comment:
This sentence is bit hard to read, I think it's caused by the combination of all the following words in a single sentence:
"is that", "so fast that", "you wouldn't expect", "from switching to", "unless", "at least as fast"